webapp/components/generic/SearchPost/SearchPost.spec.js

```diff
@@ -0,0 +1,62 @@
+import { shallowMount } from '@vue/test-utils'
+import SearchPost from './SearchPost.vue'
+
+const localVue = global.localVue
+localVue.filter('dateTime', d => d)
+
+describe('SearchPost.vue', () => {
+  let mocks, wrapper, propsData
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(string => string),
+    }
+    propsData = {
+      option: {
+        title: 'Post Title',
+        commentsCount: 3,
+        shoutedCount: 6,
+        createdAt: '23.08.2019',
+        author: {
+          name: 'Post Author',
+        },
+      },
+    }
+    wrapper = Wrapper()
+  })
+
+  const Wrapper = () => {
+    return shallowMount(SearchPost, { mocks, localVue, propsData })
+  }
+
+  describe('mount', () => {
+    it('renders post title', () => {
+      expect(wrapper.find('.search-option-label').text()).toMatch('Post Title')
+    })
+    it('renders post commentsCount', () => {
+      expect(
+        wrapper
+          .find('.search-post-meta')
+          .findAll('span')
+          .at(0)
+          .text(),
+      ).toMatch('3')
```

Comment:
I think we can improve these tests by making them more resilient... if a `span` is added these tests will potentially fail
I think we can filter the spans for the one that has text `3`, then assert that `commentsCount.exists().toBe(true)`.
I think that's an improvement, don't you?
Or did you try that and it didn't work for some reason?